sda-commons-server-openapi/src/main/java/org/sdase/commons/server/openapi/OpenApiBundle.java

```diff
@@ -219,13 +219,6 @@ default FinalBuilder withExistingOpenAPIFromClasspathResource(String path) {
   }
 
   public interface FinalBuilder extends InitialBuilder {
-    /**
-     * Disables automatic addition of the embed query parameter if embeddable resources are
-     * discovered.
-     *
-     * @return the builder.
-     */
-    FinalBuilder disableEmbedParameter();
```

Comment:
It would be nice to have this change in a separate commit with the correct `feat(server-openapi)` scope. Or can we leave it as-is @SDA-SE/sda-commons-maintainer?

Reply:
Would be nice to see it in an extra commit. Last request for change, promise! 😃

Reply:
done :)